core/src/commonMain/kotlin/com/powersync/db/PowerSyncDatabaseImpl.kt

```diff
@@ -233,8 +238,9 @@ internal class PowerSyncDatabaseImpl(
     override fun <RowType : Any> watch(
         sql: String,
         parameters: List<Any?>?,
+        throttleMs: Long?,
```

Comment:
FWIW I prefer using a `Duration` type on languages that have them in their stdlib, but it looks like we're already using milliseconds as `Long` elsewhere and it's probably better to be consistent.

Reply:
I agree, this is for consistency.